Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/ARKitRemote/ARKITREMOTE.txt

```diff
@@ -0,0 +1,27 @@
+Unity ARKit Remote
```

Comment:
What is the source of this? It feels like adding a dependency to MRTK.

We should not be redistributing this. It looks, from a cursory search, to be something that can be acquired from the Unity asset store